ccm_web/client/src/components/CreateSelectSectionWidget.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { makeStyles, Tooltip, Typography } from '@material-ui/core'
+import HelpOutline from '@material-ui/icons/HelpOutline'
+
+import CreateSectionWidget from './CreateSectionWidget'
+import SectionSelectorWidget, { SelectableCanvasCourseSection } from './SectionSelectorWidget'
+import { CanvasCourseBase, CanvasCourseSection } from '../models/canvas'
+
+const useStyles = makeStyles((theme) => ({
+  spacing: {
+    marginBottom: theme.spacing(2)
+  },
+  createSectionWidget: {
+    width: '500px'
+  },
+  sectionSelectionContainer: {
+    textAlign: 'center',
+    maxHeight: '400px'
+  },
+  newSectionHint: {
+    display: 'flex'
+  },
+  tooltip: {
+    marginLeft: theme.spacing(1)
+  },
+  createSectionContainer: {
+    paddingBottom: theme.spacing(2)
+  }
+}))
+
+interface CreateSelectSectionWidgetProps {
+  course: CanvasCourseBase
+  sections: SelectableCanvasCourseSection[]
+  selectedSection?: SelectableCanvasCourseSection
+  setSelectedSection: (section: SelectableCanvasCourseSection) => void
+  onSectionCreated: (section: CanvasCourseSection) => void
+}
+
+export default function CreateSelectSectionWidget (props: CreateSelectSectionWidgetProps): JSX.Element {
+  const classes = useStyles()
+  return (
+    <>
+      <div className={classes.createSectionContainer}>
+        <div className={`${classes.newSectionHint} ${classes.spacing}`}>
+          <Typography>Create a new section to add users</Typography>
+          <Tooltip className={classes.tooltip} placement='top' title='Enter a distinct name for this section'>
```

Comment:
Maybe we should revisit the helper text here. I'd replace "distinct" with "unique" or explain it differently.

Reply:
@pushyamig, thoughts on this?

Reply:
Keep it as it is. I don't have any opinion.